### DISCUSSION TITLE
After connection, the screen display content is misaligned or abnormal.

### DISCUSSION THREAD

**Discussion by Aya-Tatemaru** [rustdesk/rustdesk#6627] [Q&A]
**Wrong description**
When connecting to a specified device remotely, the remote desktop cannot be displayed normally. But the corresponding position of the mouse is normal. Detailed pictures are below↓↓↓↓
![bug1](https://github.com/rustdesk/rustdesk/assets/89122155/3e612339-0cb3-406d-927b-dea6a05ee77b)

**How to Reproduce**
Use any terminal device to connect to the specified device.

**Operating system**
Windows 11 Enterprise 64 Bit >> Ubuntu 22.04.3 LTS(GNOME Version 42.9)
IOS 17.1.2(21B101) >> Ubuntu 22.04.3 LTS(GNOME Version 42.9)

**RustDesk version**
Windows 11 Enterprise 64 Bit (1.2.3 2023-10-13 10:57)
iOS 17.1.2 (1.2.2)
Ubuntu 22.04.3 LTS(1.2.3 2023-10-13 11:30)

**Comment by 21pages** [comment]
Wayland or x11 ? Could you show the quality monitor? May be mistaken RGBA or BGRA, and screen size is wrong.
![2470ca654aacd8fea48d18c9496ff94](https://github.com/rustdesk/rustdesk/assets/14891774/5425ce32-3768-42d0-8b72-e499175fe47a)

**Reply by Aya-Tatemaru** [reply]
Of course, I will provide you with the information you need.
![image](https://github.com/rustdesk/rustdesk/assets/89122155/190991fc-6390-4c04-a8ad-25336546d414)

**Reply by 21pages** [reply]
I'll send you a program with the relevant logs soon.

**Reply by Aya-Tatemaru** [reply]
Do I need to provide you with my email address to receive documents? If so, I'll provide it for you.

**Reply by 21pages** [reply]
No, I'll paste the program link on this discussion, it takes about half an hour.

**Reply by 21pages** [reply]
https://github.com/21pages/rustdesk/releases/download/nightly/rustdesk-1.2.4-x86_64.deb
log path: `~/.local/share/logs/RustDesk/server/rustdesk_rCURRENT.log`
And what's your screen resolution?

**Reply by Aya-Tatemaru** [reply]
The device screen display resolution is 1900*600.

**Reply by Aya-Tatemaru** [reply]
I did not find the corresponding file in the file directory you pointed out.

/home/steamdb/.local/share/logs/RustDesk#ll
total 52
drwxrwxr-x 4 steamdb steamdb  4096 Dec  6 20:55 ./
drwxrwxr-x 3 steamdb steamdb  4096 Nov 16 09:13 ../
drwxrwxr-x 2 steamdb steamdb  4096 Dec  6 20:55 check-hwcodec-config/
drwxrwxr-x 2 steamdb steamdb  4096 Dec  6 20:55 cm/
-rw-rw-r-- 1 steamdb steamdb 10109 Nov 16 09:26 rustdesk_r2023-11-16_09-13-51.log
-rw-rw-r-- 1 steamdb steamdb  7035 Nov 16 09:30 rustdesk_r2023-11-16_09-27-46.log
-rw-rw-r-- 1 steamdb steamdb  3849 Dec  6 18:54 rustdesk_r2023-12-06_18-51-26.log
-rw-rw-r-- 1 steamdb steamdb  4688 Dec  6 19:40 rustdesk_r2023-12-06_19-00-49.log
-rw-rw-r-- 1 steamdb steamdb  3147 Dec  6 20:55 rustdesk_rCURRENT.log

**Reply by 21pages** [reply]
Ubuntu as the controlled side, and there is no  `~/.local/share/logs/RustDesk/server` directory ?

**Reply by 21pages** [reply]
Can you check if there is a `rustdesk --server` process? And paste the file `~/.local/share/logs/RustDesk/rustdesk_rCURRENT.log` if the file content contains `convert_to_yuv`.

**Reply by Aya-Tatemaru** [reply]
I ran the query but it does not contain the specified keyword. I don't know what I messed up.

**Reply by 21pages** [reply]
Can ubuntu be controlled?

**Reply by Aya-Tatemaru** [reply]
Yes, it runs on vultr. I can control it through the novnc web interface.

**Reply by 21pages** [reply]
Can you connect to it with rustdesk?

**Reply by Aya-Tatemaru** [reply]
Sure, but it has problems, and that's the problem that's currently being solved.
I don't know what I messed up. 
I'm in the process of redeploying the graphical interface and hope that the related issues can be taken care of.

**Reply by 21pages** [reply]
After connecting, there should be a file containing `convert_to_yuv` in directory `~/.local/share/logs/RustDesk`

**Reply by Aya-Tatemaru** [reply]
After I redeployed the GUI it worked, the screen issues were resolved and I'm dealing with the pop-ups.
![image](https://github.com/rustdesk/rustdesk/assets/89122155/0600c88c-a6dc-4b51-abf8-77394416c5a9)

**I don't know how it worked back, hope this information helps someone with the same problem:**
sudo apt-get remove gnome-shell
sudo apt-get purge gnome
sudo apt-get autoclean
sudo apt-get remove xfce4
sudo apt-get remove xfce4*
sudo apt-get remove kubuntu-desktop
sudo apt-get purge gnome
sudo apt-get autoclean
sudo apt-get autoremove
userdel username(userdel -r username)
reboot
sudo useradd -d /home/newusername -m newusername
sudo passwd newusername
usermod -a -G sudo newusername
sudo apt install ubuntu-desktop -y
sudo systemctl set-default graphical.target
reboot

**Reply by Aya-Tatemaru** [reply]
@21pages 
Oh no, it had the same problem again, but I found the relevant log files.
This log file is at:/root/.local/share/logs/RustDesk/server/rustdesk_rCURRENT.log

[2023-12-06 22:55:51.849495 +08:00] INFO [libs/scrap/src/common/convert.rs:230] convert_to_yuv src_pixfmt:BGRA, src_stride:[4096], src_width:1024, src_height:768, dst_fmt:EncodeYuvFormat { pixfmt: I420, w: 1024, h: 768, stride: [1024, 512, 512, 1024], u: 786432, v: 983040 }

![1701874093831](https://github.com/rustdesk/rustdesk/assets/89122155/f099997f-f898-4500-807a-20b272a2611b)

[rustdesk_rCURRENT.log](https://github.com/rustdesk/rustdesk/files/13587600/rustdesk_rCURRENT.log)

[rustdesk_r2023-12-06_22-55-19.log](https://github.com/rustdesk/rustdesk/files/13587711/rustdesk_r2023-12-06_22-55-19.log)

**Reply by 21pages** [reply]
https://github.com/21pages/test/releases/download/test/screenshot
This program will create a screenshot png on the same directory, please show the log and see if the png is normal, ignore the coredump when the program ends.
Please also try another desktop protocol, x11 or wayland.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/6628

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/6116 similar, but seems different.

**Comment by xldistance** [comment]
I'm also experiencing this issue on Ubuntu 22.04 with rustdesk 1.2.3 and 1.2.4!

**Reply by xldistance** [reply]
I'm using the xubuntu desktop

**Reply by rustdesk** [reply]
Please follow https://github.com/rustdesk/rustdesk/discussions/6627#discussioncomment-7782839 and report here.

**Comment by xldistance** [comment]
[rustdesk_rCURRENT.log](https://github.com/rustdesk/rustdesk/files/14341525/rustdesk_rCURRENT.log)
@rustdesk This is the running log of the programme

**Reply by rustdesk** [reply]
@21pages follow up

**Reply by 21pages** [reply]
Could you try https://github.com/21pages/test/releases/download/test/screenshot? This program will generate a png file, and we can judge whether the screenshot is normal based on the png and the terminal log. (Please ignore the crash)

![image](https://github.com/rustdesk/rustdesk/assets/14891774/875bb3bb-47b0-45d7-b568-956dfaf7873d)

**Reply by xldistance** [reply]
I ran screenshot without any output.
![QQ截图20240220205326](https://github.com/rustdesk/rustdesk/assets/29418474/c67fe6d7-4b1f-40e8-aa80-ee30849bbd65)

**Reply by 21pages** [reply]
Try remove test, run `./screenshot` directly

**Comment by xldistance** [comment]
@rustdesk Here's the image I generated by running screenshot straight away
```
(base) root@xldistance:/home# ./screenshot
is_x11:true
(0, 0) 1600 900
Captured data len: 5760000, Saving...
Image saved to `screenshot0_1.png`.
Captured data len: 5760000, stride: [6400], width: 1600, height: 900, format: BGRA Saving...
段错误 (核心已转储)

```
![screenshot0_1](https://github.com/rustdesk/rustdesk/assets/29418474/ca3c98c3-05f0-4043-835f-c6a43d5ac6f4)

**Reply by 21pages** [reply]
Is the physical resolution of your computer 1600*900? Is the scaling set? If you switch to wayland, will RustDesk display normal (the screenshot bin may not work in wayland)?

**Reply by xldistance** [reply]
I'm connecting to the cloud server remotely, tried multiple resolutions and it still doesn't work, switching to wayland gives me the same problem!

**Reply by xldistance** [reply]
Anydesk has the same problem, only teamviewer displays properly!

**Reply by 21pages** [reply]
It should be a screenshot problem. We can see if other open source screenshot software can work normally. If it can work normally, compare the screenshot code.
Could you try  the following? All these work on my ubuntu 22.04
1. https://trac.ffmpeg.org/wiki/Capture/Desktop with x11 desktop?
```
sudo apt install ffmpeg
ffmpeg -video_size 1600x900 -framerate 25 -f x11grab -i :0.0+0,0 output.mp4
ctrl + c
```
2. https://github.com/ImageMagick/ImageMagick
```
sudo apt install ImageMagick
import screen.png
```
3. https://shutter-project.org/downloads/third-party-packages/
```
sudo apt-get install shutter
shutter
```
4. https://gitlab.gnome.org/GNOME/gnome-screenshot
```
sudo apt install gnome-screenshot
gnome-screenshot
The file is saved to $HOME/Pictures
```
5. https://github.com/flameshot-org/flameshot
`sudo apt-get install flameshot`
6. https://github.com/ksnip/ksnip
`sudo apt install ksnip`
7. https://github.com/GNOME/gimp
```
sudo add-apt-repository ppa:otto-kesselgulasch/gimp
sudo apt update
gimp
sudo apt install gimp
File->Create->Screenshot
```
8. https://screencloud.net/
```
snap install screencloud
```

**Reply by xldistance** [reply]
Running ffmpeg and shutter is fine!

**Reply by rustdesk** [reply]
Thanks, we will modify our code following ffmpeg and shutter.

**Reply by xldistance** [reply]
gnome-screenshot failed to run
(base) root@xldistance:/home# gnome-screenshot
** Message: 10:31:39.279: Unable to use GNOME Shell's builtin screenshot interface, resorting to fallback X11.
The gimp is working fine.

**Reply by strive1216** [reply]
I'm in the same situation. Has this problem been resolved? Or is there a temporary solution?

**Reply by rustdesk** [reply]
@21pages

**Reply by rustdesk** [reply]
@strive1216 neverthless, please test our nightly build first. https://github.com/rustdesk/rustdesk/releases/tag/nightly

**Reply by 21pages** [reply]
@xldistance Could you contact me with cGFnZXMyMUAxNjMuY29t(base64)?

**Comment by 21pages** [comment]
@Aya-Tatemaru @xldistance @strive1216 If it was deployed remotely, can you tell me how it was deployed so that I can reproduce it in the same environment, including the server provider, server type, display deployment? I deployed one on Vultr using Ubuntu Desktop 22.04 Gnome. switched to Xorg, didn't reproduce it.

**Reply by strive1216** [reply]
I am using a [Tencent Cloud](https://www.tencentcloud.com/products/cvm) cloud server, with the Debian 12.4 image selected, and manually installed Xfce desktop environment.

**Reply by 21pages** [reply]
Thanks, I can reproduce this with tencent debian 12.0, xfce4.

**Reply by strive1216** [reply]
> Thanks, I can reproduce this with tencent debian 12.0, xfce4.

Hello! Has this issue been resolved?

**Comment by rustdesk** [comment]
close as fix merged https://github.com/rustdesk/rustdesk/pull/8580

**Comment by rustdesk** [comment]
Ready in nightly build, 1.2.7

**Comment by damnms** [comment]
Having the same problem, its nearly impossible to work on the remote side. Heavy green fractals with 1.2.7 on two machines with debian testing and xfce. Had to use now teamviewer :/

**Reply by rustdesk** [reply]
@21pages

**Reply by rustdesk** [reply]
Maybe there are more pixel formats

**Reply by 21pages** [reply]
Could you tell your environment? Run ffmpeg command below and show the full output, it'll show the pixfmt, usually not bgra

https://trac.ffmpeg.org/wiki/Capture/Desktop with x11 desktop?
```
sudo apt install ffmpeg
ffmpeg -video_size 1600x900 -framerate 25 -f x11grab -i :0.0+0,0 output.mp4
ctrl + c
```

**Reply by damnms** [reply]
```
root@DESKTOP-RA0V02P:/home/oli# ffmpeg -video_size 1600x900 -framerate 25 -f x11grab -i :0.0+0,0 output.mp4
ffmpeg version 7.0.1-5 Copyright (c) 2000-2024 the FFmpeg developers
  built with gcc 14 (Debian 14.1.0-5)
  configuration: --prefix=/usr --extra-version=5 --toolchain=hardened --libdir=/usr/lib/x86_64-linux-gnu --incdir=/usr/include/x86_64-linux-gnu --arch=amd64 --enable-gpl --disable-stripping --disable-libmfx --disable-omx --enable-gnutls --enable-libaom --enable-libass --enable-libbs2b --enable-libcdio --enable-libcodec2 --enable-libdav1d --enable-libflite --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-libglslang --enable-libgme --enable-libgsm --enable-libharfbuzz --enable-libmp3lame --enable-libmysofa --enable-libopenjpeg --enable-libopenmpt --enable-libopus --enable-librubberband --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libtheora --enable-libtwolame --enable-libvidstab --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx265 --enable-libxml2 --enable-libxvid --enable-libzimg --enable-openal --enable-opencl --enable-opengl --disable-sndio --enable-libvpl --enable-libdc1394 --enable-libdrm --enable-libiec61883 --enable-chromaprint --enable-frei0r --enable-ladspa --enable-libbluray --enable-libcaca --enable-libdvdnav --enable-libdvdread --enable-libjack --enable-libpulse --enable-librabbitmq --enable-librist --enable-libsrt --enable-libssh --enable-libsvtav1 --enable-libx264 --enable-libzmq --enable-libzvbi --enable-lv2 --enable-sdl2 --enable-libplacebo --enable-librav1e --enable-pocketsphinx --enable-librsvg --enable-libjxl --enable-shared
  libavutil      59.  8.100 / 59.  8.100
  libavcodec     61.  3.100 / 61.  3.100
  libavformat    61.  1.100 / 61.  1.100
  libavdevice    61.  1.100 / 61.  1.100
  libavfilter    10.  1.100 / 10.  1.100
  libswscale      8.  1.100 /  8.  1.100
  libswresample   5.  1.100 /  5.  1.100
  libpostproc    58.  1.100 / 58.  1.100
[x11grab @ 0x55e2972fbac0] Stream #0: not enough frames to estimate rate; consider increasing probesize
Input #0, x11grab, from ':0.0+0,0':
  Duration: N/A, start: 1722869061.696347, bitrate: 1152000 kb/s
  Stream #0:0: Video: rawvideo (BGR[0] / 0x524742), bgr0, 1600x900, 1152000 kb/s, 25 fps, 1000k tbr, 1000k tbn
Stream mapping:
  Stream #0:0 -> #0:0 (rawvideo (native) -> h264 (libx264))
Press [q] to stop, [?] for help
[libx264 @ 0x55e297303700] using cpu capabilities: MMX2 SSE2Fast SSSE3 SSE4.2 AVX FMA3 BMI2 AVX2 AVX512
[libx264 @ 0x55e297303700] profile High 4:4:4 Predictive, level 4.0, 4:4:4, 8-bit
[libx264 @ 0x55e297303700] 264 - core 164 r3108 31e19f9 - H.264/MPEG-4 AVC codec - Copyleft 2003-2023 - http://www.videolan.org/x264.html - options: cabac=1 ref=3 deblock=1:0:0 analyse=0x3:0x113 me=hex subme=7 psy=1 psy_rd=1.00:0.00 mixed_ref=1 me_range=16 chroma_me=1 trellis=1 8x8dct=1 cqm=0 deadzone=21,11 fast_pskip=1 chroma_qp_offset=4 threads=28 lookahead_threads=4 sliced_threads=0 nr=0 decimate=1 interlaced=0 bluray_compat=0 constrained_intra=0 bframes=3 b_pyramid=2 b_adapt=1 b_bias=0 direct=1 weightb=1 open_gop=0 weightp=2 keyint=250 keyint_min=25 scenecut=40 intra_refresh=0 rc_lookahead=40 rc=crf mbtree=1 crf=23.0 qcomp=0.60 qpmin=0 qpmax=69 qpstep=4 ip_ratio=1.40 aq=1:1.00
Output #0, mp4, to 'output.mp4':
  Metadata:
    encoder         : Lavf61.1.100
  Stream #0:0: Video: h264 (avc1 / 0x31637661), yuv444p(progressive), 1600x900, q=2-31, 25 fps, 12800 tbn
      Metadata:
        encoder         : Lavc61.3.100 libx264
      Side data:
        cpb: bitrate max/min/avg: 0/0/0 buffer size: 0 vbv_delay: N/A
frame=    4 fps=1.3 q=28.0 size=       0KiB time=00:00:00.08 bitrate=   4.8kbits[out#0/mp4 @ 0x55e297305080] video:108KiB audio:0KiB subtitle:0KiB other streams:0KiB global headers:0KiB muxing overhead: 1.727277%
frame=   88 fps= 25 q=-1.0 Lsize=     110KiB time=00:00:03.44 bitrate= 261.2kbits/s speed=0.968x    
[libx264 @ 0x55e297303700] frame I:1     Avg QP:15.15  size: 88104
[libx264 @ 0x55e297303700] frame P:22    Avg QP:13.55  size:   697
[libx264 @ 0x55e297303700] frame B:65    Avg QP:23.34  size:    97
[libx264 @ 0x55e297303700] consecutive B-frames:  1.1%  0.0%  3.4% 95.5%
[libx264 @ 0x55e297303700] mb I  I16..4: 42.0% 30.5% 27.5%
[libx264 @ 0x55e297303700] mb P  I16..4:  0.1%  0.1%  0.0%  P16..4:  3.3%  0.3%  0.2%  0.0%  0.0%    skip:96.1%
[libx264 @ 0x55e297303700] mb B  I16..4:  0.0%  0.0%  0.0%  B16..8:  0.9%  0.0%  0.0%  direct: 0.0%  skip:99.1%  L0:42.7% L1:57.3% BI: 0.0%
[libx264 @ 0x55e297303700] 8x8 transform intra:32.5% inter:96.4%
[libx264 @ 0x55e297303700] coded y,u,v intra: 26.3% 14.0% 14.3% inter: 0.1% 0.2% 0.2%
[libx264 @ 0x55e297303700] i16 v,h,dc,p: 52% 42%  2%  4%
[libx264 @ 0x55e297303700] i8 v,h,dc,ddl,ddr,vr,hd,vl,hu: 28% 16% 32% 11%  1%  1%  1%  4%  6%
[libx264 @ 0x55e297303700] i4 v,h,dc,ddl,ddr,vr,hd,vl,hu: 37% 30% 13%  5%  3%  2%  4%  3%  3%
[libx264 @ 0x55e297303700] Weighted P-Frames: Y:0.0% UV:0.0%
[libx264 @ 0x55e297303700] ref P L0: 90.9%  0.4%  5.4%  3.3%
[libx264 @ 0x55e297303700] ref B L0: 13.6% 86.3%  0.1%
[libx264 @ 0x55e297303700] ref B L1: 95.2%  4.8%
[libx264 @ 0x55e297303700] kb/s:249.35
Exiting normally, received signal 2.

```

**Reply by 21pages** [reply]
Could you make a screenshot of the abnormal display?

**Reply by damnms** [reply]
```
tream #0:0 -> #0:0 (rawvideo (native) -> h264 (libx264))
Press [q] to stop, [?] for help
[libx264 @ 0x5578d46d4a80] using cpu capabilities: MMX2 SSE2Fast SSSE3 SSE4.2 AVX FMA3 BMI2 AVX2
[libx264 @ 0x5578d46d4a80] profile High 4:4:4 Predictive, level 4.0, 4:4:4, 8-bit
[libx264 @ 0x5578d46d4a80] 264 - core 164 r3095 baee400 - H.264/MPEG-4 AVC codec - Copyleft 2003-2022 - http://www.videolan.org/x264.html - options: cabac=1 ref=3 deblock=1:0:0 analyse=0x3:0x113 me=hex subme=7 psy=1 psy_rd=1.00:0.00 mixed_ref=1 me_range=16 chroma_me=1 trellis=1 8x8dct=1 cqm=0 deadzone=21,11 fast_pskip=1 chroma_qp_offset=4 threads=12 lookahead_threads=2 sliced_threads=0 nr=0 decimate=1 interlaced=0 bluray_compat=0 constrained_intra=0 bframes=3 b_pyramid=2 b_adapt=1 b_bias=0 direct=1 weightb=1 open_gop=0 weightp=2 keyint=250 keyint_min=25 scenecut=40 intra_refresh=0 rc_lookahead=40 rc=crf mbtree=1 crf=23.0 qcomp=0.60 qpmin=0 qpmax=69 qpstep=4 ip_ratio=1.40 aq=1:1.00
Output #0, mp4, to 'output.mp4':
  Metadata:
    encoder         : Lavf59.27.100
  Stream #0:0: Video: h264 (avc1 / 0x31637661), yuv444p(tv, progressive), 1600x900, q=2-31, 25 fps, 12800 tbn
    Metadata:
      encoder         : Lavc59.37.100 libx264
    Side data:
      cpb: bitrate max/min/avg: 0/0/0 buffer size: 0 vbv_delay: N/A
frame=    1 fps=0.0 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=N/Aframe=   15 fps=0.0 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   frame=   28 fps= 27 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   frame=   41 fps= 26 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   frame=   54 fps= 26 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   frame=   67 fps= 26 q=28.0 size=       0kB time=00:00:00.32 bitrate=   1.2kbits/frame=   80 fps= 26 q=28.0 size=       0kB time=00:00:00.84 bitrate=   0.5kbits/frame=   93 fps= 26 q=28.0 size=       0kB time=00:00:01.36 bitrate=   0.3kbits/frame=  106 fps= 26 q=28.0 size=       0kB time=00:00:01.88 bitrate=   0.2kbits/frame=  109 fps= 24 q=-1.0 Lsize=     159kB time=00:00:04.24 bitrate= 306.7kbits/s speed=0.95x    
video:157kB audio:0kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 1.351579%
[libx264 @ 0x5578d46d4a80] frame I:1     Avg QP:21.02  size: 79925
[libx264 @ 0x5578d46d4a80] frame P:28    Avg QP:15.83  size:  1936
[libx264 @ 0x5578d46d4a80] frame B:80    Avg QP:27.90  size:   320
[libx264 @ 0x5578d46d4a80] consecutive B-frames:  0.9%  3.7%  0.0% 95.4%
[libx264 @ 0x5578d46d4a80] mb I  I16..4: 29.2% 44.8% 26.0%
[libx264 @ 0x5578d46d4a80] mb P  I16..4:  0.2%  0.2%  0.4%  P16..4:  1.0%  0.1%  0.1%  0.0%  0.0%    skip:98.1%
[libx264 @ 0x5578d46d4a80] mb B  I16..4:  0.0%  0.2%  0.1%  B16..8:  0.7%  0.0%  0.0%  direct: 0.0%  skip:99.0%  L0:34.9% L1:64.2% BI: 0.9%
[libx264 @ 0x5578d46d4a80] 8x8 transform intra:44.8% inter:51.0%
[libx264 @ 0x5578d46d4a80] coded y,u,v intra: 18.7% 4.5% 3.2% inter: 0.1% 0.0% 0.0%
[libx264 @ 0x5578d46d4a80] i16 v,h,dc,p: 28% 71%  0%  1%
[libx264 @ 0x5578d46d4a80] i8 v,h,dc,ddl,ddr,vr,hd,vl,hu: 18% 10% 71%  2%  0%  0%  0%  0%  0%
[libx264 @ 0x5578d46d4a80] i4 v,h,dc,ddl,ddr,vr,hd,vl,hu: 43% 24% 15%  3%  3%  3%  3%  3%  3%
[libx264 @ 0x5578d46d4a80] Weighted P-Frames: Y:0.0% UV:0.0%
[libx264 @ 0x5578d46d4a80] ref P L0: 87.3%  5.7%  6.2%  0.9%
[libx264 @ 0x5578d46d4a80] ref B L0: 74.8% 23.9%  1.2%
[libx264 @ 0x5578d46d4a80] ref B L1: 97.1%  2.9%
[libx264 @ 0x5578d46d4a80] kb/s:293.06
Exiting normally, received signal 2.
root@debian:/home/gabi# ^C
root@debian:/home/gabi# 
```

**Reply by damnms** [reply]
https://ibb.co/xmnKWHf 
thats all i was able to capture from the green/invisible terminal
i (oli) try to connect to her (gabi) workstation

**Reply by 21pages** [reply]
@damnms My computer is bgr0 too, bgr0 means that the tranparency is not set or undefined, could you run the following command and send `screenshot.bgra` to my email? `c3VuYm9lYXN5QGdtYWlsLmNvbQo=`, base64.  And please tell your resolution.

Or you can open it with bin editor to see whether tranparency is always 0xFF
![1722908136531](https://github.com/user-attachments/assets/8c9fa822-873f-43ee-9ffd-54090f241c77)

https://github.com/21pages/rustdesk/releases/download/tmp/screenshot (ignore the crash at the end)

**Reply by damnms** [reply]
not sure what command i should run? you mean the attached binary? and should i run it on my workstation or on the machine where i try to connect to? resolution on the client should be fullhd, on my workstation its wqhd

**Reply by rustdesk** [reply]
Run this file https://github.com/21pages/rustdesk/releases/download/tmp/screenshot  on the controlled machine ( A->B, B is the controlled machine) which has the problem.

**Reply by damnms** [reply]
i will see when i can provide it. currently the machine is unavailable (its about 200km away and the person is not there)

**Reply by rustdesk** [reply]
Thanks

**Reply by damnms** [reply]
I was able to get that file, sent it to you, thanks for checking it.

**Reply by 21pages** [reply]
@damnms The transparency is 0xFF too, what's the resolution of the controlled side?

**Reply by damnms** [reply]
it *should* be fullhd (1920 x 1080) but i will check that today

**Reply by damnms** [reply]
the resolution is 2048x1152. i changed it now to fullhd (1920x1080) but somehow i am now unable to connect. it says waiting for remote display or so (in german it is "Warte auf anderen Bildschirm ...")

**Reply by rustdesk** [reply]
Some ref:
- https://github.com/TigerVNC/tigervnc/issues/684#issuecomment-495410781 create color profile for vnc server
- https://code.videolan.org/videolan/vlc/-/merge_requests/4336
